lib/metasploit/framework/data_service/proxy/loot_data_proxy.rb

```diff
@@ -4,7 +4,7 @@ def report_loot(opts)
     begin
       self.data_service_operation do |data_service|
         if !data_service.is_a?(Msf::DBManager)
-          opts[:data] = Base64.urlsafe_encode64(opts[:data]) if opts[:data]
+          opts[:data] = Base64.urlsafe_encode64(opts[:data].empty? ? "" : opts[:data].join('')) if opts[:data] and opts[:data].kind_of?(Array) else opts[:data]
```

Comment:
@phra how do I test this change? I tested without this change and it still works fine.

Reply:
I tried the password prompt module on catalina: it's passing a string to this function that expect an array.